pydicom/data/base.py

```diff
@@ -0,0 +1,71 @@
+"""pydicom data manager"""
+#
+# Copyright (c) 2008-2012 Darcy Mason
+# Copyright (c) 2017 pydicom AUTHORS
+# This file is part of pydicom, released under a modified MIT license.
+#    See the file LICENSE included with this distribution, also
+#    available at https://github.com/pydicom/pydicom
+#
+
+import fnmatch
+import os
+
+
+def get_datadir():
+    '''get the data directory base
+    '''
+    return os.path.abspath(os.path.dirname(__file__))
+
+
+def get_dataset(dataset=None, pattern=None, return_base=False):
```

Comment:
this function depending of `return_base` outputs a directory or returns a list of files, doesn't it? 
If so, then this is a problem.
posible solutions:
a) if pattern is none, `get_dataset` returns a list with all the elements in the directory
b) you create a separate function to get the base like `get_testdata_home`. (remember that the name **dataset** was agreed to not be used in this sense in #388 )